core/src/cuda_runtime.rs

```diff
@@ -230,23 +228,19 @@ impl<T: Clone> PinnedVec<T> {
     }
 
     fn check_ptr(&mut self, _old_ptr: *mut T, _old_capacity: usize, _from: &'static str) {
-        #[cfg(feature = "cuda")]
```

Comment:
@sakridge this looking totally broken right? This fn would have been doing nothing for the past couple of weeks

Reply:
Pinning is disabled recently anyway. Checking the api flag here might save some work in the non-pinning case.

Reply:
Ok I see, I'll add the api check here.